### ISSUE TITLE
Whether to add a mask to the sky in the mipnerf360 dataset?

### ISSUE THREAD

**Issue by OUC-voyage** [kcheng1021/GaussianPro#72]
For the mipnerf360 data set, did the author add a mask to the sky of the scene? The depth of the sky is often estimated inaccurately. The sky is easy to adopt a progressive propagation strategy, which easily generates a large number of Gaussians, which has an impact on the final depth and normals.

**Comment by kcheng1021** [comment]
Yes, the sky needs mask. The sky is close to infinitely distant, with almost no parallax in the image, and propagation cannot estimate the correct depth.

**Comment by wln19** [comment]
> Mipnerf360数据集,笔者是不是在场景的天空中加了一个面具? 天空的深度经常被错误地估计。天易于采用渐进传播策略,容易产生大量高斯,对最终深度和法线有影响。
I would like to ask if you have successfully added the mask to the Minpnerf dataset?

**Comment by OUC-voyage** [comment]
> Yes, the sky needs mask. The sky is close to infinitely distant, with almost no parallax in the image, and propagation cannot estimate the correct depth.

Is the mipnerf360 dataset downsampled by 4× for outdoor scenes and 2× for indoor scenes, or are all images uniformly resized to 1.6k resolution?
